engine/src/test/resources/log4j.properties

```diff
@@ -17,28 +17,25 @@
 # under the License.
 #
 
-log4j.rootLogger=INFO,CONSOLE
+# Set root logger level to WARN to not flood build logs
+# set manually to DEBUG or INFO for debugging purposes
+log4j.rootLogger=warn, root
 
-log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
-log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
-log4j.appender.CONSOLE.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c{2} %M - %m%n
+# Redirect root appender to standard error
+log4j.appender.root=org.apache.log4j.ConsoleAppender
+log4j.appender.root.target=System.err
+log4j.appender.root.layout=org.apache.log4j.PatternLayout
+log4j.appender.root.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c{2} %M - %m%n
 
-log4j.appender.malhar=org.apache.log4j.RollingFileAppender
-log4j.appender.malhar.layout=org.apache.log4j.PatternLayout
-log4j.appender.malhar.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c{2} %M - %m%n
-#log4j.appender.malhar.File=/tmp/app.log
+log4j.appender.apex=org.apache.log4j.ConsoleAppender
+log4j.appender.apex.threshold=info
```

Comment:
Why a separate threshold specification for this appender?

Reply:
It prevents DEBUG and TRACE messages from LoggerUtilTest logger. I can change the console appender threshold inside the test only if this is a concern.

Reply:
Yes I think that would be better.

Reply:
Please explain why will it be better and what is the concern. IMO, it will complicate test code.

Reply:
These are common engine test log4j properties applicable to all tests and apex is a generic appender for org.apache.apex.*, so I don't think it should have specific properties for a single test. Also, multiple appenders are not being used yet there are two log levels one debug at the logger level and another being info at appender level. It makes debug redundant.